aslam_cv_common/include/aslam/common/sensor.h

```diff
@@ -50,7 +50,10 @@ class Sensor : public YamlFileSerializable {
   virtual Sensor::Ptr cloneAsSensor() const = 0;
 
   // Set and get the sensor id.
-  void setId(const SensorId& id);
```

Comment:
any particular reason for this change?

Reply:
Just consistency because the setters and getters were spread all around since marius did some after me and we happened to do it differently. Not important, but I noticed and decided to put them all in the same place.